src/shared/remoteSourceTrackingService.ts

```diff
@@ -196,7 +199,7 @@ export class RemoteSourceTrackingService extends ConfigFile<RemoteSourceTracking
     // any item in an aura/LWC bundle needs to represent the top (bundle) level and the file itself
     // so we de-dupe via a set
     Array.from(new Set(elements.flatMap((element) => getMetadataKeyFromFileResponse(element)))).map((metadataKey) => {
-      const revision = revisions[metadataKey];
+      const revision = revisions[decodeURI(metadataKey)];
```

Comment:
This is the real change, the other changes are just about ordering in the file after removing the eslint rule